### ISSUE TITLE
Crashes when playing content

### ISSUE THREAD

**Issue by Goblinlordx** [jellyfin/jellyfin-androidtv#1361]
### Describe the bug

1. Goto Movies
2. Select a movie
3. Play movie
4. After playing for a few minutes, the app crashes and restarts

In addition to the crash, the movie is not displayed in the "continue watching" area. This seems to occur with 1080p content but not 720p content. It might be worth mentioning that I can play the same videos on jellyfin in chrome and they work fine. I will try to isolate the differences between what works and doesn't and try to grab logs when I can.

### Application version

0.12.3

### Where did you install the app from?

Google Play

### Device information

Chromecast with Google TV

### Android version

Android 10

### Jellyfin server version

10.7.7

**Comment by mueslimak3r** [comment]
Is this when playing media that transcodes, direct plays, or both? You can find that info in the server dashboard when the video is playing.

**Comment by Goblinlordx** [comment]
@mueslimak3r I will check when I get home this evening. I am fairly certain it is a transcode though. standby and I will check though (~7-12h from now).

**Comment by 0x6b-dev** [comment]
I seem to be having the same issue. Looks like transcoding on 1080p content.

App appears to be hard crashing instead of gracefully dialling down the quality or something.

**Comment by Goblinlordx** [comment]
@0x6b-dev Just FYI, you should be able to surround the log with 3 backticks ("`" character is usually on same key as tilde ~ on keyboard) before and after to make it formatted in a code block 👍 

```
should result in a block
like this which is a bit easier to read for
stuff like this
```

**Comment by 0x6b-dev** [comment]
> log with 3 backticks

Apologies im kinda new to this, there is a backtick there - it just isn't displaying properly. Will fiddle with it.

**Comment by Goblinlordx** [comment]
@0x6b-dev All good 👍

**Comment by Goblinlordx** [comment]
Seems I was incorrect, it was a direct play. It gets about 4:40-5:05 or so (varies) and then crashes. Video details (ffprobe):

Video 1 (crashes):
```
Stream #0:0(und): Video: h264 (High) (avc1 / 0x31637661), yuv420p, 1920x800 [SAR 1:1 DAR 12:5], 2249 kb/s, 23.98 fps, 23.98 tbr, 24k tbn, 47.95 tbc (default)
Stream #0:1(und): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, 5.1, fltp, 384 kb/s (default)
```

Video 2 (crashes):
```
Stream #0:0(und): Video: h264 (High) (avc1 / 0x31637661), yuv420p, 1920x800 [SAR 1:1 DAR 12:5], 2048 kb/s, 23.98 fps, 23.98 tbr, 13978 tbn, 47.95 tbc (default)
Stream #0:1(und): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 134 kb/s (default)
```

Video 3 (plays fine 2:30:00+):
```
Stream #0:0(und): Video: h264 (High) (avc1 / 0x31637661), yuv420p, 1280x528 [SAR 1:1 DAR 80:33], 772 kb/s, 23.98 fps, 23.98 tbr, 96k tbn, 47.95 tbc (default)
Stream #0:1(und): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 95 kb/s (default)
```

**Comment by mueslimak3r** [comment]
A fix is being iterated on that addresses these crashes and others related to failing to start playback. It doesn't fix the server-side `Object reference not set to an instance of an object`, but it does allow for graceful recovery.
#1360

**Comment by Goblinlordx** [comment]
This does appear to be fixed in `0.13.0.beta.1`. I am not sure if it is related but I am now getting pauses on some content. I will try to isolate if it is specific types or anything. It's hard to tell right now and I don't see any logs (server or client) indicating any issues.

**Comment by mueslimak3r** [comment]
> This does appear to be fixed in `v0.13.0-beta.1`. I am not sure if it is related but I am now getting pauses on some content. I will try to isolate if it is specific types or anything. It's hard to tell right now and I don't see any logs (server or client) indicating any issues.

This sounds like buffering (I assume the pausing is temporary). It could be buffering due to your networking, or it could be the app handling buffering sub-optimally.

Please open a new issue for the pausing/buffering, and if you're satisfied that this issue (the crashes) has been resolved, feel free to close it.

**Comment by Goblinlordx** [comment]
I think this one is closed as it doesn't seem to be crashing. I did find logs saying it is buffering. It is hard to justify that it should be though as I have been trying to test via different methods and hadn't had issues with buffering until now. Anyways, I think that is a separate issue. Thanks~ Closing as resolved in `v0.13.0-beta.1`

**Comment by Goblinlordx** [comment]
As a note, I was able to confirm it was of course network issues. For some reason... my chromecast with google tv has a terrible network connection =/... still working on figuring that out/resolving it but for now it works well on a different android tv device 👍
